doc/gpu/index.rst

```diff
@@ -4,7 +4,7 @@ XGBoost GPU Support
 
 This page contains information about GPU algorithms supported in XGBoost.
 
-.. note:: CUDA 12.0, Compute Capability 5.0 required (See `this list <https://en.wikipedia.org/wiki/CUDA#GPUs_supported>`_ to look up compute capability of your GPU card.)
+.. note:: CUDA 12.0, Compute Capability 7.0 required (See `this list <https://developer.nvidia.com/cuda-gpus>`_ to look up compute capability of your GPU card.)
```

Comment:
Actually no. Thinking more about it, we can stay with 5.0. It's just not in the default compilation targets. I don't think we need to state the sm in doc as we simply support what CUDA supports. This SM limitation applies to the binary distribution on PyPI and maven, it's not applicable to source build and other distribution methods like conda.